composer.json

```diff
@@ -80,6 +80,7 @@
         "cebe/indent": "~1.0.2",
         "dealerdirect/phpcodesniffer-composer-installer": "*",
         "dms/phpunit-arraysubset-asserts": "^0.5",
+        "php-forge/foxy": "^0.1",
```

Comment:
We're currently using asset packagist in the apps:

https://github.com/yiisoft/yii2-app-basic/blob/master/composer.json#L70

Does it work well together?

Reply:
This is branch 22, asset-packagist does not exist here, it has been removed.